index.rst

```diff
@@ -13,13 +13,57 @@
 
-Add content here
-================
+Introduction
+============
+
+The May 2023 Commissioning Science Validation Bootcamp presented a wonderful introduction to the existing metrics and visualization tools already available in the LSST Science Pipelines.  The work on calibration production and verification has not previously used these tools, but given their advanced state, switching now before the full camera comes online is a priority.  This should hopefully resolve some of the difficulty of managing and visualizing the results that are already sufficiently complicated for just the LATISS data.  The goal of this technote is define the additional tasks that are necessary to add to the [``cp_pipe``](https://github.com/lsst/cp_pipe) and [``cp_verify``](https://github.com/lsst/cp_verify) packages to take advantage of these tools, to identify any functionality missing from ``analysis_tools`` and ``Sasquatch``, and to present a roadmap for this process.
+
+``cp_verify`` currently generates a series of plots and image views in a series of notebooks.  This is reasonably convenient for an individual (or small group) checking the quality of calibrations for a single detector, but as either the number of interested parties or detectors grow, this becomes more difficult.  We have also wanted a way to present results from the calibration construction and verification similar to the "run reports" the Camera Team produces from their [``eo_pipe``](https://github.com/lsst-camera-dh/eo_pipe) based code.  The plotting ability of ``analysis_tools`` (and the associated Plot Navigator) present an excellent way to present the things currently found in the notebooks, and by turning the numerical results into ``Metrics``, we can also track these values over time via ``Sasquatch``.
+
+
+Calibration Products Improvements
+=================================
+
+Metrics
+-------
+
+The ``cp_verify`` package should organize the quality test results (as defined in DMTN-101) that are being calculated into formal ``Metrics``.  As it is unlikely that there is an amplifier level dimension, these probably need to report the minimum, maximum, and median values found per detector, as well as the number of amplifiers that failed the test requirements.  As the variety of most of these tests is fairly limited, defining these metrics does not seem like a problem.  Generating these metrics can be added to the existing ``MergeTasks``, at the expense of ensuring that ``analysis_tools`` is a prerequisite for ``cp_verify``.  This seems like the obvious first step for this integration.
```

Comment:
Won't we need a detector dimension?  For example, an amp with a developing bright defect shouldn't condemn the rest of the detector.   How hard would be this?